01-html/03-caitlinandcollin.html

```diff
@@ -2,4 +2,228 @@
 I designed and built this site for my friends!
 Pretty straightforward layout. The trick on this one is going to be the nav bar that sticks at the top of the screen after you scroll.
 That functionality is handled via javascript, but I want to see how you will roll out the HTML anyhow.
--->
\ No newline at end of file
+-->
+
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Collin & Caitlin</title>
+</head>
+
+<body>
+  <header>
+    <hgroup>
+      <h1>Collin</h1>
+      <h1>-&-</h1>
+      <h1>Caitlin</h1>
```

Comment:
I like the thought of using the `hgroup` tag, but it's important to think of how your html might look to a robot. A robot will look for a single `h1` tag to find the title of the page, with the desired result being "Collin & Caitlin". Using `span`s within a single `h1` would be appropriate here.